server/run.go

```diff
@@ -71,13 +72,15 @@ func Run(cfg config.ShipperConfig) error {
 		return fmt.Errorf("couldn't create queue: %w", err)
 	}
 
+	// Make a placeholder console output to read the queue's events
+	out := output.NewConsoleOutput(queue)
```

Comment:
nit: `output.NewConsoleOutput` could just be `output.NewConsole`. 

I doubt the output package structure is going to stay the same as we add more of them though so tidying this might be pointless now.